.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+3.0.5
```

Comment:
This is the only change that isn't just coming from upstream. As you can see in the Dockerfile, they're on Ruby 3.1.3.